exporters/prometheus/CMakeLists.txt

```diff
@@ -15,16 +15,13 @@
 include_directories(include)
 find_package(prometheus-cpp CONFIG REQUIRED)
 
-add_library(prometheus_exporter_deprecated src/prometheus_collector.cc
```

Comment:
The target is removed here, but I didn't see it is added back in other places but it is referenced below in `target_include_directories`.

Reply:
fixed